exercises/complete/Makefile

```diff
@@ -6,11 +6,14 @@ verify:
 
 execute: | execute_cs execute_go execute_java execute_js execute_py
 
+compile: | compile_cs compile_go compile_js compile_java compile_py
```

Comment:
If we're splitting up compilation from execution anyway, how about we use the new `dafny build` top-level command?

```
compile_cs:
        dafny build \
	src/Index.dfy \
	--target:cs \
	/out:runtimes/AwsKmsArn
	
execute_cs: compile_cs
        dotnet \
	runtimes/AwsKmsArn.dll \
	arn:aws-cn:kms:us-west-2:658956600833:key/b3537ef1-d8dc-4780-9f5a-55776cbb2f7f \
	arn:aws-cn:kms:us-west-2:658956600833:b3537ef1-d8dc-4780-9f5a-55776cbb2f7f \
        ...
```

IMO that helps emphasize that Dafny can really produce something reasonable in many different target languages from the same source. It also avoids the awkward `--args` arguments. :)

If you like this I'm happy to push a commit that does this for all target langauges.

Reply:
Yes, please, this is MUCH nicer

Reply:
Done, just watch out for https://github.com/dafny-lang/dafny/issues/2867